modules/hostap/src/supp_api.h

```diff
@@ -156,6 +156,15 @@ int supplicant_set_power_save(const struct device *dev, struct wifi_ps_params *p
  */
 int supplicant_set_twt(const struct device *dev, struct wifi_twt_params *params);
 
+/** Set rts threshold
+ *
+ * @param dev Pointer to the device structure for the driver instance.
+ * @param rts_threshold rts threshold to set
+ *
+ * @return 0 if ok, < 0 if error
+ */
+int supplicant_set_rts_threshold(const struct device *dev, unsigned int rts_threshold);
```

Comment:
Please remove this duplicate `supplicant_set_rts_threshold` in this header file.

Reply:
Removed